README.md

```diff
@@ -443,6 +443,7 @@ Table of Contents
   * [codecov.io](https://codecov.io/) — Code coverage tool (SaaS), free for Open Source and 1 free private repo
   * [CodeFactor](https://www.codefactor.io) — Automated Code Review for Git. Free version includes unlimited users, unlimited public repositories and 1 private repo.
   * [codescene.io](https://codescene.io/) - CodeScene prioritizes technical debt based on how the developers work with the code and visualizes organizational factors like team coupling and system mastery. Free for Open Source.
+  * [Codiga](https://www.codiga.io/) - Codiga continously analyzes your code, finds code issues in many domains (security, safety, performance, code style, etc) and automates your code reviews.
```

Comment:
Pls add something like "Free for small teams and open source projects"

Reply:
ah thank you @scorpio18 and thank you @juli1 for the contribution but I'll close this one then